### PR TITLE
Add Cookies

### PR DESCRIPTION
**Pull request by Elchi3**
I wonder if https://github.com/web-platform-dx/web-features/blob/main/features/cookie-enabled.yml should be part of this?

### CODE REVIEW

**Comment by ddbeck** [comment]
> I wonder if https://github.com/web-platform-dx/web-features/blob/main/features/cookie-enabled.yml should be part of this?

`cookieEnabled` only just recently became interoperable, so let's leave it where it is for now. https://github.com/web-platform-dx/web-features/blob/main/features/cookie-enabled.yml.dist#L6

File: features/cookies.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
-draft_date: 2024-10-21
-name: HTTP State Management Mechanism
-description: TODO
+name: Cookies
+description: Cookies store limited amounts of data and remember state information.
```
Comment (ddbeck) [inline features/cookies.yml:2]: I think this is a good start. Maybe we can call out the actual entry point into this (I didn't mention `document.cookie`, though we could) and elaborating on where the state resides?

```suggestion
description: The `Set-Cookie` HTTP header stores limited amounts of data that persist across request and response, creating shared state between the browser and server.
```

**Review by ddbeck — APPROVED** [review]
This is pretty good—I have one optional suggestion for the description.

Also now that I've reviewed this, I'm a little more open to the idea of merging `cookieEnabled` into this (though I could go either way on it). Unfortunately, doing so would be a breaking release; we could do that (we've got some other possible breakages to make), but if you think it's the right thing to do, please file and issue so we can prep to do that.

This is ready to merge, once you've had a think about my description suggestion. Thank you!

**Comment by Elchi3** [comment]
Again I think it would be better if had a rule about when to merge things back into the "parent" feature. Without such rule, we are again making a judgment call. That said, I'm going to leave things as is and hope that future selves will come up with something that will make this consistent throughout the data.

**Pull request merged**